libs/dh/shared/feature-flags/src/lib/feature-flags.ts

```diff
@@ -41,6 +41,10 @@ export const dhFeatureFlagsConfig = {
       DhAppEnvironment.test_002,
     ],
   },
+  'calculations-include-all-grid-areas': {
+    created: '18-03-2024',
+    disabledEnvironments: [DhAppEnvironment.test_001],
```

Comment:
issue: the PR description says "a feature flag so it could only apply to test-001" but doesn't the code above do the opposite - it disables the feature in `test_001` and enables it everywhere else 😊